### DISCUSSION TITLE
Is this hardware supported?

### DISCUSSION THREAD

**Discussion by kevev** [SmoothWAN/SmoothWAN#166] [General & Help]
https://www.waveshare.com/wiki/CM4-DUAL-ETH-BASE

It uses rPi Compute 4 module.

Thanks in advance!  :)

**Comment by TalalMash** [comment]
The hardware shouldn't be an issue, but I haven't gotten enough feedback about the CM4 being supported, all I know that the previous SmoothWAN versions didn't work, while the latest does according to one report since the latest use owrt22.03 which supports cm4. I highly recommend getting any generic mini x86/PC instead (intel atom) considering it would cost the same and much more reliable.
